Support/Testing/Patches/Disable-broken-Concurrent-tests.patch

```diff
@@ -1,253 +1,12 @@
-diff --git a/test/functionalities/thread/concurrent_events/TestConcurrentEvents.py b/test/functionalities/thread/concurrent_events/TestConcurrentEvents.py
```

Comment:
Aha, makes sense.

On Mon, Jun 6, 2016 at 4:24 PM, Francis Ricci <notifications@github.com>
wrote:

> In Support/Testing/Patches/Disable-broken-Concurrent-tests.patch
> <https://github.com/facebook/ds2/pull/301#discussion_r65987170>:
>
> > @@ -1,253 +1,12 @@
> > -diff --git a/test/functionalities/thread/concurrent_events/TestConcurrentEvents.py b/test/functionalities/thread/concurrent_events/TestConcurrentEvents.py
>
> The new patch disables the whole suite of Concurrent tests (so smaller
> because it doesn't do each one separately).
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/301/files/143d344a4af0c6611f39ca185d5ea36a9cc34542..330c6506268dc34745ce6feda71036f43efd114d#r65987170>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbGvjXhh9qOgviJQ2L-bkVqBr0h07ks5qJKw9gaJpZM4IQI8S>
> .
>



-- 
Stephane Sezer